loader/src/main/java/net/neoforged/fml/loading/FMLConfig.java

```diff
@@ -103,6 +112,17 @@ private static Object maxThreads(final Object value) {
         for (ConfigValue cv : ConfigValue.values()) {
             cv.buildConfigEntry(configSpec, configComments);
         }
+
+        // Make sure that we don't end up "correcting" the config and removing dependency overrides
+        // Since we're not writing them by default, the default value can be null and we accept any objects (parsing and validation is done when the config is loaded)
+        configSpec.define("dependencyOverrides", Map::of, object -> true);
```

Comment:
The `Map::of` is actually useless and should be replaced again by `() -> null`. Sorry 😅